### DISCUSSION TITLE
Font choice and fall back secondary font settings

### DISCUSSION THREAD

**Discussion by XDTZ** [standardnotes/forum#2690] [Suggestions]
Example captured on Android Studio
![Example](http://i.imgur.com/CmV1yXQ.png)

Basically it's necessary in order to customize the look of eastern Asian language mixed with normal ASCII characters. I wish you would consider implementing this feature.

**Comment by JaspalSuri** [comment]
Thanks for the feedback; I've passed it along to the team. I'm sorry about any issues caused by not having this.

**Comment by arielsvg** [comment]
Hey @XDTZ, we don't currently offer font customization, however if there is no good fallback font for a language you are writing in then we should fix that 🙂 For example the next version of the app will release with a better simplified Chinese font on Windows.
Can you tell us what languages you're writing in? Perhaps include a screenshot?
